libr/core/cbin.c

```diff
@@ -2903,62 +2903,127 @@ static int bin_mem(RCore *r, int mode) {
 	return true;
 }
 
-static void bin_pe_versioninfo(RCore *r) {
+static void bin_pe_versioninfo(RCore *r, int mode) {
 	Sdb *sdb = NULL;
 	int num_version = 0;
 	int num_stringtable = 0;
 	int num_string = 0;
 	const char *format_version = "bin/cur/info/vs_version_info/VS_VERSIONINFO%d";
 	const char *format_stringtable = "%s/string_file_info/stringtable%d";
 	const char *format_string = "%s/string%d";
-	r_cons_printf ("=== VS_VERSIONINFO ===\n\n");
+	if (IS_MODE_JSON (mode)) {
+		r_cons_printf ("[");
+	} else {
+		r_cons_printf ("=== VS_VERSIONINFO ===\n\n");
+	}
+	bool firstit_dowhile = true;
```

Comment:
Is there something more consistent with the rest of the codebase for doing this same thing? Feels a bit cheesy but I didn't know how to accomplish different behaviours on first/last iterations otherwise.